trie/hexary.py

```diff
@@ -182,33 +181,28 @@ def get_proof(self, key):
         node = self.get_node(self.root_hash)
         trie_key = bytes_to_nibbles(key)
 
-        return self._get_proof(node, trie_key)
+        if node == BLANK_NODE:
+            return (node,)
 
-    @staticmethod
-    def _missing_key_error(key_nibbles):
-        raise KeyError("Key %s does not exist" % nibbles_to_bytes(key_nibbles))
+        return self._get_proof(node, trie_key)
 
     def _get_proof(self, node, trie_key, proven_len=0, last_proof=tuple()):
         updated_proof = last_proof + (node, )
         unproven_key = trie_key[proven_len:]
 
         node_type = get_node_type(node)
         if node_type == NODE_TYPE_BLANK:
-            raise self._missing_key_error(trie_key)
+            return last_proof
```

Comment:
If this is changed to `updated_proof` instead, then it will include the blank node in the proof.